R/mod-panel-section.R

```diff
@@ -119,6 +123,15 @@ mod_panel_section_server <- function(input, output, session, synapse, syn, user,
       )
     )
   })
+  
+  observeEvent(input$overall_score, {
+    if (!is.na(input$overall_score)) {
+      if (input$overall_score < 0 || input$overall_score > 1) {
+        runjs("alert('Enter a valid Final Overall Score value ranging from 0 to 1.');")
+        updateNumericInput(session, "overall_score", value = 0)
+      }
```

Comment:
Not a shiny expert here, but does setting min/max negate the need for this?

Reply:
So, the only real benefit i think this has is that, if a value is inputted outside the range, rather than rejecting it and making it null, it sets it back to zero... but maybe we don't really need that.

Reply:
If this is required to surface the friendly error message, I say we keep it.